authors/apps/authentication/models.py

```diff
@@ -112,3 +118,12 @@ def get_short_name(self):
         the user's real name, we return their username instead.
         """
         return self.username
+
+    @property
+    def auth_token(self):
+        """creates a token for registration"""
+        self.encoded_jwt = jwt.encode(
+            {'name': self.username,
+             'exp': datetime.now() + timedelta(minutes=15)},
+            SECRET_KEY, algorithm='HS256').decode('utf-8')
+        return self.encoded_jwt
```

Comment:
please try using the Token from the rest_framework.authtoken.models, which is provided for you from the rest_framework